geotrek/infrastructure/models.py

```diff
@@ -123,6 +158,16 @@ class Infrastructure(MapEntityMixin, BaseInfrastructure):
     """ An infrastructure in the park, which is not of type SIGNAGE """
     type = models.ForeignKey(InfrastructureType, verbose_name=_("Type"), on_delete=models.CASCADE)
     objects = InfrastructureGISManager()
+    maintenance_difficulty = models.ForeignKey(InfrastructureMaintenanceDifficultyLevel,
+                                               verbose_name=_("Maintenance difficulty"),
+                                               help_text=_("Danger level of maintenance agents' interventions on infrastructure"),
+                                               blank=True, null=True,
```

Comment:
```suggestion
                                               blank=True, null=True,
                                               related_name='infrastructure_used'
```
pas forcement à faire maintenant mais jette un coup d'oeil sur la doc sur les related_name